model/group_impl.go

```diff
@@ -33,6 +36,49 @@ func (repo *EntRepository) CreateGroup(ctx context.Context, name string, descrip
 	return ConvertEntGroupToModelGroup(created), nil
 }
 
+func (repo *EntRepository) GetOwners(ctx context.Context, groupID uuid.UUID) ([]*Owner, error) {
+	groupowners, err := repo.client.Group.
+		Query().
+		Where(group.IDEQ(groupID)).
+		QueryOwner().
+		All(ctx)
```

Comment:
申し訳ないのですが追加でひとつ
```suggestion
	groupowners, err := repo.client.Group.
		Query().
		Where(group.IDEQ(groupID)).
		QueryOwner().
		Select(user.FieldID).
		All(ctx)
```
のようにしてIDだけ取ってくれば十分なので、この追加をお願いします:pray: